inject.js

```diff
@@ -149,18 +149,14 @@ chrome.extension.sendMessage({}, function(response) {
       this.video.classList.add('vsc-initialized');
       this.video.dataset['vscid'] = this.id;
 
-      switch (location.hostname) {
```

Comment:
Technically, I think we can do regex matching in switch. Any particular reason why you prefer if's?